content/docs/user-guide/data-management/remote-storage/index.md

```diff
@@ -0,0 +1,157 @@
+# Remote Storage
+
+_DVC remotes_ provide optional/additional storage to backup and share your data
+and ML models. For example, you can download data artifacts created by
+colleagues without spending time and resources to regenerate them locally. See
+also `dvc push` and `dvc pull`.
+
+<admon type="info">
+
+DVC remotes are similar to [Git remotes] (e.g. GitHub or GitLab hosting), but
+for <abbr>cached</abbr> data instead of code.
+
+[git remotes]: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+
+</admon>
+
+DVC does not provide or recommend a specific storage service (unlike code
+repos). You can bring your own platform from a wide variety of
+[supported storage types](#supported-storage-types).
+
+Main uses of remote storage:
+
+- Synchronize large files and directories tracked by DVC.
+- Centralize or distribute data storage for sharing and collaboration.
+- Back up different versions of datasets and models (saving space locally).
```

Comment:
IMHO this doesn't add much that isn't already said or implied above and below, so I would consider dropping it to save space.

Reply:
OK, up to you. Lmk